webapp/src/shared/components/Header/Header.tsx

```diff
@@ -35,15 +40,35 @@ type NoFigureHeader = HeaderCommon & {
 
 type HeaderProps = IconHeader | AvatarHeader | NoFigureHeader;
 
+export const calcDownwardsDisplacement = (len: number): number => {
+  switch (len) {
+    case 0:
+      return 0;
+    case 1:
+      return 0;
+    case 2:
+      return 30;
+    case 3:
+      return 34;
+    case 4:
+      return 38;
+    default:
+    case 5:
+      return 43;
+  }
```

Comment:
Well, these are experimental coefficients. I can spend several hours trying to find what's the rule, or make a table that makes the independent variable match a dependent variable that makes the thing look nice. I could name them, but it would be something along the line of "yDisplacementThatMatchesTheCaseOfFiveElementsAndMakesItLooksNice", if I'm being honest, which is a very useless name.
I could also make it look like a table, with types and coordinates, and it would look better, but it's completely indistinguishable conceptually from this implementation, as long as I name the function properly, which it is.

Reply:
Anyway, it depends on whether we want vertically displayed arrays of buttons or not.
If we don't need them, we don't need this, we're good.

Reply:
The vertical array of buttons displays properly outside of the header component, and without all this math.
I know you don't like the idea of having more than one button in the chat page, but I'm gonna need more than your personal preference to remove it.

Reply:
I made it horizontal, as [we have voted](https://42born2code.slack.com/archives/C033YQ28QA0/p1668472053117479?thread_ts=1668338365.304939&cid=C033YQ28QA0).